group_vars/matrix_servers

```diff
@@ -3642,6 +3642,11 @@ matrix_media_repo_systemd_required_services_list_auto: |
     ([devture_postgres_identifier ~ '.service'] if devture_postgres_enabled and matrix_media_repo_database_hostname == devture_postgres_connection_hostname else [])
   }}
 
+matrix_media_repo_generate_signing_key: "{{ matrix_homeserver_implementation == 'synapse' or matrix_homeserver_implementation == 'dendrite'}}"
+matrix_media_repo_homeserver_signing_key: "{{ matrix_media_repo_synapse_signing_key if matrix_homeserver_implementation == 'synapse' else (matrix_media_repo_dendrite_signing_key if matrix_homeserver_implementation == 'dendrite' else '') }}"
```

Comment:
It seems like these 2 should have definitions in `defaults/main.yml` in the role, given that they are used within the role.

`matrix_media_repo_synapse_signing_key` and `matrix_media_repo_dendrite_signing_key` seem like helper variables used by the playbook, so they're OK to remain in `group_vars/matrix_servers` only.